trendmicro_apex/icon_trendmicro_apex/connection/connection.py

```diff
@@ -0,0 +1,58 @@
+import komand
+from .schema import ConnectionSchema, Input
+# Custom imports below
+from requests.packages.urllib3 import exceptions
+from requests.exceptions import RequestException
+from komand.exceptions import ConnectionTestException
+import warnings
+import requests
+import json
+import time
+from icon_trendmicro_apex.util.util import create_base64_checksum
+from icon_trendmicro_apex.util.util import create_jwt_token
+
+warnings.simplefilter('ignore', exceptions.InsecureRequestWarning)
+
+class Connection(komand.Connection):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+        self.header_string = ''
+
+    def connect(self, params):
+        self.api_key = params.get(Input.API_KEY).get('secretKey')
+        self.application_id = params.get(Input.APPLICATION_ID).get('secretKey')
+        self.hostname = params.get(Input.HOSTNAME)
+
+    def create_jwt_token(self, api_path, http_method, request_body):
+        jwt_token = create_jwt_token(self.application_id, self.api_key, http_method, api_path, self.header_string,
+                                request_body)
+        self.jwt_token = jwt_token
+        self.header_dict = {'Authorization': 'Bearer ' + jwt_token, 'Content-Type': "application/json"}
+
+    def test(self):
+        # list UDSO's
+        json_payload = ''
+        api_path = "/WebApp/api/SuspiciousObjects/UserDefinedSO/"
+        request_url = self.hostname + api_path
+        jwt_token = self.create_jwt_token(api_path, 'GET', json_payload)
```

Comment:
create_jwt_token returns nothing, this doesn't need to be assigned to a variable.